packages/react-components/components/SimpleMessagingCard.tsx

```diff
@@ -20,11 +20,22 @@ export default function SimpleMessagingCard({
   callToActions,
   testID,
 }: Props) {
-  const icon = React.isValidElement(iconProp) ? (
-    iconProp
+  const icon = iconProp ? (
+    React.isValidElement(iconProp) ? (
+      iconProp
```

Comment:
What happens if a link breaks?

Reply:
The space allocated for the image would remain empty.